checkout/features/index.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5781626)